packages/docs/src/routes/docs/(qwikcity)/api/index.mdx

```diff
@@ -383,6 +383,53 @@ test('should render the button and navigate', async () => {
 });
 ```
 
+> If you are using `routeLoader$`s in a Qwik Component that you want to test, you can provide a `loaders` prop to mock the data returned by the loaders
+
+```ts title="src/loaders/product.loader.ts"
+import { routeLoader$ } from '@builder.io/qwik-city';
+
+export const useProductData = routeLoader$(async () => {
+  const res = await fetch('https://.../product');
+  const product = (await res.json()) as Product;
+  return product;
+});
+```
+
+```tsx title="src/components/product.tsx"
```

Comment:
nit pick but this should be a route path. Otherwise you must re-export the route loader